theme/tutorial-built-in-themes.asciidoc

```diff
@@ -73,10 +74,18 @@ public class MyTheme implements AbstractTheme {
 }
 ----
 
-For more control there is `getHeadInlineContents()` and `getBodyInlineContents()` that returns a collection of html
-that will be inlined to the BootstrapPage head. this could return for instance add a
-custom style to have the correct typography e.g.
-`<custom-style><style include="lumo-typography"></style></custom-style>`
+For more control there is `getBodyInlineContents()` that returns a collection of html
+that will be inlined to the BootstrapPage head. For instance, it can be used to add a
```

Comment:
`that will be inlined into the BootstrapPage body.`

As the method name also suggests that it is to the body.